src/mon/MonClient.cc

```diff
@@ -770,11 +769,24 @@ void MonClient::tick()
 
       send_log();
     }
+
+    _un_backoff();
```

Comment:
_start_hunting starts at the min value, and the min is applied when we un-backoff, so it should never be less than that.  This does change things to be slightly more aggressive, though, since we can tick at any point after we finally connect.  We only un-backoff after we've authenticated though.

Oh.. at least that's what I meant to do. Repushed with an update!